whisper/whisperv5/api.go

```diff
@@ -374,17 +374,17 @@ type PostArgs struct {
 	Payload  hexutil.Bytes `json:"payload"`
 	WorkTime uint32        `json:"worktime"`
 	PoW      float64       `json:"pow"`
-	FilterID uint32        `json:"filterID"`
-	PeerID   hexutil.Bytes `json:"peerID"`
+	FilterID string        `json:"filterid"`
```

Comment:
I did not realize this. Thanx for your comment!
But I will leave it like this, because it does not really matter.